source/backend/api/Services/CompReqFinancialService.cs

```diff
@@ -61,11 +61,11 @@ public IEnumerable<PimsCompReqFinancial> SearchCompensationRequisitionFinancials
             // When a contractor exports data, they should only see data for files they have been assigned to.
             if (pimsUser.IsContractor)
             {
-                return allMatchingFinancials.Where(f => f.CompensationRequisition.AcquisitionFile.PimsAcquisitionFileTeams.Any(afp => afp.PersonId == pimsUser.PersonId));
+                return allMatchingFinancials.Where(f => f.CompensationRequisition?.AcquisitionFile?.PimsAcquisitionFileTeams.Any(afp => afp.PersonId == pimsUser.PersonId) == true);
```

Comment:
we now have compensations that have no acquisition, so dereferencing like this breaks.

Reply:
logged https://jira.th.gov.bc.ca/browse/PSP-9180 to ask how contractors should be able to access lease export data.

Reply:
(since there is no lease team corresponding to the acquisition team).